lib/mayaUsd/fileio/translators/translatorCurves.cpp

```diff
@@ -99,7 +99,7 @@ bool UsdMayaTranslatorCurves::Create(
     }
 
     VtArray<GfVec3f> points;
-    VtArray<int>     curveOrder;
+    VtArray<int>     curveOrders;
     VtArray<int>     curveVertexCounts;
```

Comment:
Changed the name of the variable to be plural like the other to represent this is a vector of values.